scripts/ci/push-image.sh

```diff
@@ -83,5 +83,5 @@
 
         echo "DEBUG: Executing command:"
         echo "depot build --push --platform \"${PLATFORMS}\" --build-arg VERSION_STRING=\"${VERSION_STRING}\" ${tags} -f Dockerfile ."
-        depot build --push --platform "${PLATFORMS}" --build-arg VERSION_STRING="${VERSION_STRING}" ${tags} -f Dockerfile .
+        depot build --push --platform "${PLATFORMS}" --build-arg VERSION_STRING="${VERSION_STRING}" "${tags}" -f Dockerfile .
```

Comment:
This change will cause a failure on the command execution as it contains the flag and we should only quote the value